great_expectations/render/renderer/content_block/validation_results_table_content_block.py

```diff
@@ -356,15 +360,36 @@ def row_generator_fn(evr, styling=None, include_column_name=True):
             try:
                 unexpected_statement = cls._get_unexpected_statement(evr)
             except Exception as e:
-                logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_message = f'''\
+An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will \
+not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to \
+diagnose and repair the underlying issue.  Detailed information follows:  
+                '''
+                exception_traceback = traceback.format_exc()
+                exception_message += f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                logger.error(exception_message, e, exc_info=True)
             try:
                 unexpected_table = cls._get_unexpected_table(evr)
             except Exception as e:
-                logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_message = f'''\
+An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will \
+not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to \
+diagnose and repair the underlying issue.  Detailed information follows:  
+                '''
+                exception_traceback = traceback.format_exc()
+                exception_message += f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                logger.error(exception_message, e, exc_info=True)
             try:
                 observed_value = [cls._get_observed_value(evr)]
             except Exception as e:
-                logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_message = f'''\
+An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will \
+not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to \
+diagnose and repair the underlying issue.  Detailed information follows:  
+                '''
+                exception_traceback = traceback.format_exc()
+                exception_message += f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                logger.error(exception_message, e, exc_info=True)
```

Comment:
Tiny nitpick - these messages look identical - could they be a single variable?

Reply:
@Aylr I fixed it -- could you please take a quick look (just to be on the safe side)?  Thanks!